web/src/click_handlers.js

```diff
@@ -894,6 +893,35 @@ export function initialize() {
         }
     });
 
+    // COMPOSE BUTTON TOOLTIPS
```

Comment:
Can you explain the reasoning behind putting this in the click-handlers file instead of the compose-tooltips file? Or even the Tippy file?

Reply:
Ohh,  initially the logic I used in this PR used click handler for creating the array of tooltip instances of all the buttons. This was enhanced after review from @sayamsamal. 
Although I tried to put this in another file but for other files , the creation of tippy instances is not initiated, hence tooltips are not rendered. 
Probable reason could be that `compose_tooltip.js` file is executed before the tippy library is initiated but I am not sure. Another reason could be the trigger we provide is superimposed by some else trigger which does not create the tooltip array .
Would love to hear your opinion and possible solutions.

Reply:
Have you experimented with reordering the files so Tippy gets loaded up before `compose_tooltip.js`? That might be a worthy prep commit, given how central Tippy is to the codebase now.